doc/conf.py

```diff
@@ -316,19 +316,27 @@
 
 examples_dirs = ['../examples', '../tutorials']
 gallery_dirs = ['auto_examples', 'auto_tutorials']
-
+os.environ['_MNE_BUILDING_DOC'] = 'true'
 try:
     mlab = mne.utils._import_mlab()
     # Do not pop up any mayavi windows while running the
     # examples. These are very annoying since they steal the focus.
     mlab.options.offscreen = True
-    scrapers = ('matplotlib', 'mayavi')
 except Exception:
     scrapers = ('matplotlib',)
+    report_scraper = None
 else:
     # Let's do the same thing we do in tests: reraise traits exceptions
     from traits.api import push_exception_handler
     push_exception_handler(reraise_exceptions=True)
+    report_scraper = mne.report._ReportScraper()
+    scrapers = ('matplotlib', 'mayavi', report_scraper)
+
+
+def setup(app):
```

Comment:
out of curiosity, where is this variable `app` defined?

and why don't you have to provide the `scrapers` variable to sphinx gallery config?

Reply:
`app` is provided by Sphinx when it calls this function

`scrapers` is modified above in the `mayavi` try/except clause